saviSimulation/savi/jason_processing/SAVIWorld_model.java

```diff
@@ -36,9 +37,12 @@
 	private int UGV_SIZE;
 	private int NUMBER_UAV;
 	private int NUMBER_UGV;
+	private double SENSORS_ERROR_PROB;
+	private double SENSORS_ERROR_STD_DEV;
+	private double WIFI_ERROR_PROB;
 	/********** CONSTANTS THAT CANNOT BE LOADED FROM THE CONF FILE **********/
-	public final int X_PIXELS = 900;
-	public final int Y_PIXELS = 700;
+	public final int X_PIXELS = 874;
+	public final int Y_PIXELS = 699;
 	//int Z_PIXELS = 500;
 	String CONSOLE_ID = "console"; //not loaded from config file
```

Comment:
I think we've now introduced an ID when the object is first created (this was a quick thing to test)